docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -101,36 +78,19 @@ Host crc
     StrictHostKeyChecking no
     UserKnownHostsFile=/dev/null
 ----
++
+* Replace `<path to SSH key>` with the path to your SSH key. You must set the `IdentityFile` key in the file to get full working access to your {OpenStackShort} environment.
```

Comment:
```suggestion
* Replace `<path to SSH key>` with the path to your SSH key. You must provide a value for `IdentityFile` to get full working access to your {OpenStackShort} environment.
```
IIUC, then this might be more succinct. If I don't, then ignore :)